src/commands/create.ts

```diff
@@ -49,6 +49,7 @@ const command: GluegunCommand = {
     if (!fileCategory) {
       const informBadInput = userCategoryInput && !fileCategory;
       const { category } = await prompt.ask([questionFileCategory(informBadInput)]);
+      // @ts-ignore
```

Comment:
I check this locally - I think this is an unneeded `@ts-ignore`

Reply:
I had to add this to get it to work on my machine, not sure what's up with all that 🤷

Reply:
do you have `typescript` globally installed? That might be why - when I run `yarn tsc` no errors are reported